x-pack/plugins/siem/server/lib/ip_details/mock.ts

```diff
@@ -0,0 +1,373 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { DomainsEdges, NetworkDirectionEcs } from '../../graphql/types';
+
+import { DomainsBuckets, IpOverviewHit } from './types';
+
+export const responseAggs: IpOverviewHit = {
+  aggregations: {
+    destination: {
+      doc_count: 882307,
+      geo: {
+        doc_count: 62089,
+        results: {
+          hits: {
+            total: {
+              value: 62089,
+              relation: 'eq',
+            },
+            max_score: null,
+            hits: [
+              {
+                _source: {
+                  destination: {
+                    geo: {
+                      continent_name: 'Asia',
+                      region_iso_code: 'IN-KA',
+                      city_name: 'Bengaluru',
+                      country_iso_code: 'IN',
+                      region_name: 'Karnataka',
+                      location: {
+                        lon: 77.5833,
+                        lat: 12.9833,
+                      },
+                    },
+                  },
+                },
+                sort: [1553894176003],
+              },
+            ],
+          },
+        },
+      },
+      lastSeen: {
+        value: 1553900180003,
```

Comment:
Optional: I'd use the `new Date(....).valueOf()` trick to get a human readable string in these spots where you can.

Reply:
Just in case I need to compare it to the `value_as_string` below but it's not a biggie if you don't need/want to as I can at least trust it's the same as the thing below it.

Reply:
I'm in agreement with that technique and have benefited from our changes to it in unit/integration tests, but this mock object is as returned from Elasticsearch, so I think it may be best to leave as is in this instance. Makes it easy to swap out for new mock data, and you can compare the number `value` with that of the `sort` field as well.